starter-code/routes/drones.js

```diff
@@ -1,20 +1,27 @@
-const express = require('express');
+const express = require("express");
 
 // require the Drone model here
 
 const router = express.Router();
 
-
-router.get('/drones', (req, res, next) => {
+router.get("/drones", (req, res, next) => {
   // Iteration #2
+  Drones.find({}, (err, users) => {
+    if (err) {
+      next(err);
+    } else {
+      const data = {
+        drones: drones
+      };
+    }
+  });
 });
 
-
-router.get('/drones/new', (req, res, next) => {
+router.get("/drones/new", (req, res, next) => {
   // Iteration #3
```

Comment:
use only /new
not
/drones/new

in app.js you do "/drones" with this you do not need to repeat on all routes "/drones" in your router file for drones.js